### DISCUSSION TITLE
Double Commander 1.1.12 using alternate sort type for each path folder

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1574] [General]
Hello. Thanks for reading my topic.
I see here happening an strange behavior.
I have selected "sort by extension" for folder path view.
However when browsing that selected type is randomly reverted to "sort by name" for some folder.
Example. In sequence some folders are sorted by extensions and the next will be sorted by name and so next too.
That behavior not happen in previous version.

That is an new feature in new current version ? If yes as disable it ?

Have an nice day.

**Comment by alexx2000** [comment]
Hi, no new feature. Sorting does not depend on the current path.

**Comment by Augusto7743** [comment]
That means now need select the sort type for each path ?
In previous version if selecting "sort by type" is used for all path directories. Now is randomly if will not use that setting. Here happening very much that strange behavior. Any solution ?
I want use current version, but is being hard to use when accessing several paths. I need select sort by type almost all time.

**Reply by alexx2000** [reply]
No, it means that nothing changed. I don't know how you got this strange behaviour.

I need a detail info how to reproduce it.

**Comment by Augusto7743** [comment]
Yes .. really an strange behavior.
I select "sort by type" and work in current path.
After browse in next folder that setting works, the second and third works too ... randomly in next sequence goes back to "sort by name" and strangely next paths "sort by type" works again.
I will try empty the dcm settings folders and create an new settings files.
After I post the result here.

**Comment by Augusto7743** [comment]
In moment the only detail about that strange behavior is the change to "sort by name" in paths with file images using extension PNG (not png not sensitive).
